OtripleS.Web.Api.Tests.Unit/Services/Calendars/CalendarServiceTests.Validations.Delete.cs

```diff
@@ -0,0 +1,98 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Moq;
+using OtripleS.Web.Api.Models.Calendars;
+using OtripleS.Web.Api.Models.Calendars.Exceptions;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.Calendars
+{
+    public partial class CalendarServiceTests
+	{
+		[Fact]
+		public async Task ShouldThrowValidatonExceptionOnDeleteWhenIdIsInvalidAndLogItAsync()
+		{
+			// given
+			Guid randomCalendarId = default;
+			Guid inputCalendarId = randomCalendarId;
```

Comment:
You don't need two variables for this.
Also, the first variable isn't really random if it's specifically assigned the value of `default`.

You want to do something like this:
```csharp
Guid invalidCalendarId = Guid.Empty;
```